src/app/users/users-profile/users-profile.component.ts

```diff
@@ -6,12 +6,24 @@ import { UserService } from '../../shared/user.service';
 import { Validators } from '@angular/forms';
 import { DialogsFormService } from '../../shared/dialogs/dialogs-form.service';
 import { CustomValidators } from '../../validators/custom-validators';
-
+import { Location } from '@angular/common';
 import { switchMap } from 'rxjs/operators';
 import { of } from 'rxjs/observable/of';
 
 @Component({
-  templateUrl: './users-profile.component.html'
+  templateUrl: './users-profile.component.html',
+  styles: [ `
+    .space-container {
+      margin: 64px 30px;
+    }
+    .view-container {
+      background-color:#fff;
+      padding: 8%;
+    }
+    mat-list-item h3{
```

Comment:
Please create a `.primary-color` class in `styles.scss` which uses the `$primary` variable so if we change our color scheme this will change with it.